responsiveness.css

```diff
@@ -0,0 +1,229 @@
+@media (min-width: 1020px) {
+  .section {
+    padding-left: 2%;
+    padding-right: 2%;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile {
+    display: none;
+    visibility: hidden;
+  }
+
+  .heading {
+    font-weight: 800;
+    font-size: 43px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    width: 96%;
+  }
+
+  .seebutton {
+    display: none;
+  }
+
+  .desktop {
+    visibility: visible;
+  }
+
+  .head-desktop-bg {
+    background-image: url(./desktop-bg.png);
+    background-size: 100%;
+    background-repeat: no-repeat;
+  }
+
+  .navbar-link {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    margin-left: 20%;
+    margin-right: 32%;
+    position: sticky;
+  }
+
+  .navbar-link a {
+    text-decoration: none;
+    color: #344563;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+  }
+
+  .header-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 21px;
+    width: 2%;
+  }
+
+  .flexcontainer {
+    width: 60%;
+    margin-top: 8%;
+    margin-left: 18%;
+    margin-bottom: 38%;
+    background-image: none;
+  }
+
+  .paragraph {
+    width: 84%;
+    margin-top: -5%;
+    margin-left: 6%;
+  }
+
+  .second {
+    margin-left: 8%;
+    margin-bottom: -5%;
+  }
+
+  .grid-container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    width: 45%;
+    margin-left: 12%;
+    margin-bottom: 15%;
+    gap: 3%;
+  }
+
```

Comment:
According to the project requirement, you are to ensure that your design is the same as the Figma design given. Please Your Project section for the desktop version should have  2 rows with 3 projects each (6 projects in total) to give you a better view of what we have in the template also remember to follow all alignments for this section 😄 .

Figma design:

https://github.com/tajulafreen/Portfolio/assets/20537712/35b83063-4487-42fb-a667-cbd68caf92ff



Your design:

https://github.com/tajulafreen/Portfolio/assets/20537712/935a843a-fbb6-4383-a012-7fd63218e594